src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialog.tsx

```diff
@@ -115,7 +115,9 @@ const CreatePaymentDialog = ({
     colonyData?.processedColony?.isWhitelistActivated;
 
   const filteredVerifiedRecipients = useMemo(() => {
-    return isWhitelistActivated
+    return isWhitelistActivated &&
+      colonyData?.processedColony?.whitelistedAddresses &&
+      colonyData?.processedColony?.whitelistedAddresses?.length > 0
```

Comment:
Just a nitpick, but if `colonyData?.processedColony?.whitelistedAddresses` evaluates, then you no longer need the check on the length, since you know it exists already.

So that would make it something like:
```suggestion
    return isWhitelistActivated &&
      colonyData?.processedColony?.whitelistedAddresses &&
      colonyData.processedColony.whitelistedAddresses.length > 0
```

Reply:
Good point, removing the redundancy.